mlir/test/Dialect/Func/annotate-types.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: mlir-opt %s --split-input-file --annotate-function-type | FileCheck %s
+
+// CHECK-LABEL: func.func @one_arg(%arg0: tensor<f32> {func.orig_type = tensor<f32>}) -> (tensor<f32> {func.orig_type = tensor<f32>}) {
+func.func @one_arg(%arg0 : tensor<f32>) -> tensor<f32> {
+  return %arg0 : tensor<f32>
+}
+
+
+// -----
```

Comment:
```suggestion
// -----

```

Reply:
In the rest of LLVM, it looks like the splits leave one line above and one line below each test case.